pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -706,6 +741,30 @@ class AnalyticsImpl implements Analytics {
     _surveyHandler.dismiss(survey, false);
     send(Event.surveyShown(surveyId: survey.uniqueId));
   }
+
```

Comment:
We are able to replace the functionality provided by `ErrorHandler` with this instance method instead.

Instead of having the error events get sent as soon we encounter them, we now keep track of them and send them when we invoke `Analytics.close`.